### PR TITLE
Add Honeycomb Sink for Heaptser

### PR DESCRIPTION
**Pull request by systemizer**
Honeycomb is a logging service that aggregates logs from multiple sources. This PR introduces a new heapster sink that allows Honeycomb users to access heapster metrics.

### CODE REVIEW

**Comment by DirectXMan12** [comment]
I want to make this clear: if you add a sink, you're on the hook for maintaining it.  If there are issues, they'll be assigned to you, and it's your responsibility to deal with them in a timely fashion.  If issues are assigned to you and not dealt with, or if there are dependency issues that aren't dealt with, we (the Heapster maintainers) may remove the sink.

I don't mean to sound harsh, but I'd like to avoid maintenance issues in the future.

If you'd like to go forward with this, add yourself to the sink-owners.md file as part of this PR.

**Comment by emfree** [comment]
Thanks @DirectXMan12! I've updated sink-owners.md in this PR. (I'm responsible for Kubernetes integrations at honeycomb, and worked on this patch with @systemizer.)

File: docs/sink-owners.md [hunk 1]
```diff
@@ -37,3 +37,4 @@ List of Owners
 | Graphite        | :heavy_check_mark: | :x:                | @jsoriano / @theairkit                        | :new: #1341    |
 | Wavefront       | :heavy_check_mark: | :x:                | @ezeev                                        | :ok:           |
 | Librato         | :heavy_check_mark: | :x:                | @johanneswuerbach                             | :ok:           |
+| Honeycomb       | :heavy_check_mark: | :heavy_check_mark: | @emfree                                       | :new: #1762    |
```
Comment (DirectXMan12) [inline docs/sink-owners.md]: this doesn't need to marked as new unless you plan to do follow-up PRs.

**Comment by DirectXMan12** [comment]
This has passed tests, so I'm going to go ahead an merge.

**Comment by emfree** [comment]
Wonderful, thank you!! :tada: :tada:

**Pull request merged**